examples/python/lookersdk-flask/app/__init__.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from flask import Flask
```

Comment:
[the goog](https://g3doc.corp.google.com/eng/doc/devguide/py/style/index.md?cl=head#s2.2-imports) wants it like this: `import flask` (see suggested change below)